src/snowflake/snowpark/modin/plugin/_internal/binary_op_utils.py

```diff
@@ -236,10 +237,34 @@ def _compute_subtraction_between_snowpark_timestamp_columns(
         "rmul": "mul",
         "rsub": "sub",
         "rmod": "mod",
+        "__rand__": "__and__",
+        "__ror__": "__or__",
     }
 )
 
 
+def _op_is_between_two_timedeltas_or_timedelta_and_null(
```

Comment:
```suggestion
def _is_timedelta_op(
```
nit

Reply:
I actually prefer the original name `_op_is_between_two_timedeltas_or_timedelta_and_null` as I think it's clearer. `_is_timedelta_op` sounds ambiguous to me since it could be include an op between a Timedelta and a datetime which is not what we want here.

Reply:
SG